CODEOWNERS

```diff
@@ -1 +1 @@
-*       @agunde406 @eloaverona @jsmitchell @ltseeley @peterschwarz @rberg2 @rbuysse @ryanlassigbanks @shannynalayna @vaporos
+*       @agunde406 @eloaverona @jsmitchell @ltseeley @peterschwarz @rberg2 @rbuysse @ryanlassigbanks @shannynalayna @vaporos @txross
```

Comment:
Please exclude the codeowner/maintainer changes -- there is a separate process for adding maintainers which requires voting, etc.

Reply:
Acknowledged. I will omit changes to those files in the next iteration.